dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -702,9 +669,13 @@ private boolean isContent(String id, boolean byInode, long langId, boolean respe
 		}
 
 		return map;
-
 	}
-
-
-
+	
+	private boolean ifIdentifierIsUnpublishedOrExpired (Date publishOrExpireDate, Date tmDate ){
+	    if(UtilMethods.isSet(publishOrExpireDate) && publishOrExpireDate.before(tmDate)){

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1126)